PROFILER.md

```diff
@@ -28,6 +28,19 @@ environment variables:
   * The default is 1,000 microseconds (1ms).
   * `IOPIPE_PROFILER_SAMPLERATE` as an environment variable
 
+Additionally the following are advanced environment variables which the
+profiler uses to slightly change its behavior, these are not intended for
+normal users to use unless in specific cases.
+
+ * `IOPIPE_PROFILER_LOCAL_DUMP_PATH`
```

Comment:
Include the default values for these vars in the docs

Reply:
The default value for these variables is `null`, but I can say that there is no assigned value (if it is not set then no file will be generated).